openpype/hosts/blender/scripts/build_workfile.py

```diff
@@ -376,6 +376,49 @@ def build_anim(project_name, asset_name):
     )
 
 
+def build_lipsync(project_name, shot_name):
+    """Build lipsync workfile.
+
+    Args:
+        project_name (str):  The current project name from OpenPype Session.
+        asset_name (str):  The current asset name from OpenPype Session.
+    """
+    # Check if kitsu_module is available
+    kitsu_module = ModulesManager().modules_by_name.get("kitsu")
+    if not kitsu_module or not kitsu_module.enabled:
+        return
+
+    import gazu
+
+    # Connect to gazu
+    gazu.client.set_host(os.environ["KITSU_SERVER"])
+    gazu.log_in(os.environ["KITSU_LOGIN"], os.environ["KITSU_PWD"])
+
+    # Get casting
```

Comment:
Can't you use the exact same code from `build_layout` and make it a common function?